Source/FolioReaderCenter.swift

```diff
@@ -157,10 +157,6 @@ open class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UIColl
         enableScrollBetweenChapters(scrollEnabled: true)
         view.addSubview(collectionView)
 
-        if #available(iOS 11.0, *) {
-            collectionView.contentInsetAdjustmentBehavior = .never
```

Comment:
This is a fix for a bug on iOS 11 collectionView scroll, why did you remove?

Reply:
Oh, sory sorry, i reverted incorrect commit.
Fixed! https://github.com/FolioReader/FolioReaderKit/pull/342/commits/658e0bac08ec3b27f97ea361ea6f1d6f375aaec5